frontend/app/templates/project/show/transportation/existing-conditions.hbs

```diff
@@ -1,16 +1,9 @@
 {{! transportation/existing-conditions controller template }}
-<div class="row">
-  <div class="sixteen wide column">
-    <Transportation::StudyAreaMap
-      @analysis={{model.transportationAnalysis}}
-    />
-  </div>
+<div class="sixteen wide column">
```

Comment:
this column div seems redundant since the elements in the ExistingConditionsSteps component are wrapped in divs with `class="sixteen wide column"`. not sure it matters

Reply:
Hmm, you mean inside the `ExistingConditionsSteps` template themselves?